src/cookies.rs

```diff
@@ -108,21 +110,19 @@ impl Client {
 
     /// Delete a single cookie from the current document.
     ///
-    /// See [16.4 Delete Cookie](https://www.w3.org/TR/webdriver1/#delete-cookie) of the
+    /// See [16.4 Delete Cookie](https://w3.org/TR/webdriver1/#delete-cookie) of the
     /// WebDriver standard.
     pub async fn delete_cookie(&mut self, name: &str) -> Result<(), error::CmdError> {
         self.issue(WebDriverCommand::DeleteCookie(name.to_string()))
             .await
-            .map(|_| ())
+            .map(drop)
```

Comment:
```suggestion
            .map(|_| ())
```
I don't think `map(drop)` is meaningfully clearer. Did clippy complain?

Reply:
It turns out to be a [quite subjective thing](https://github.com/rust-lang/rust-clippy/issues/2523). For me it's clearer, shorter and complies more with lints like [`clippy::redundant_closure`](https://rust-lang.github.io/rust-clippy/master/#redundant_closure), and we do use such style a lot. Though, I cannot recall Clippy explicitly complaining on it at the moment.

If you're not comfortable with it, I'm OK to revert.

Reply:
Heh, yeah, I'm definitely more on the "drop is special in Rust and shouldn't be mixed with 'ignore' even if the outcome is the same" side of that discussion. Thanks for reverting.